uint8 a
uint8 b
uint8 c
uint8 d
uint8 e
uint8 f
uint8 g
uint8 h
uint8 i
uint8 j
uint8 k
uint8 l
uint8 m
uint8 n
uint8 o
uint8 p
uint8 q
uint8 r
uint8 s
uint8 t
uint8 u
uint8 v
uint8 w
uint8 x
uint8 y
uint8 z
uint8 num_0
uint8 num_1
uint8 num_2
uint8 num_3
uint8 num_4
uint8 num_5
uint8 num_6
uint8 num_7
uint8 num_8
uint8 num_9
uint8 left_shift
uint8 right_shift
uint8 left_ctrl
uint8 right_ctrl
uint8 left_alt
uint8 right_alt
uint8 space
uint8 enter
uint8 backspace
uint8 tab
uint8 escape
uint8 dele
uint8 insert
uint8 home
uint8 end
uint8 page_up
uint8 page_down
uint8 arrow_up
uint8 arrow_down
uint8 arrow_left
uint8 arrow_right
